frontend/ui/screensaver.lua

```diff
@@ -23,6 +23,7 @@ local logger = require("logger")
 local _ = require("gettext")
 local ffiUtil = require("ffi/util")
 local T = ffiUtil.template
+local util = require("util")
```

Comment:
Pssht should go up a bit (ffiUtil, lfs, logger, util, _, T) if I'm not mistaken.